godel_msgs/srv/TrajectoryPlanning.srv

```diff
@@ -1,13 +1,29 @@
 # The cartesian points representing path to be blended
 ProcessPath path
 
-#extra parameters that tell descartes how to behave
+## extra parameters that tell descartes how to behave
+# frames
 string group_name
 string tool_frame
 string world_frame
+
+# as opposed to a profilimeter raster path
+bool is_blending_path
+
+# controls whether or not the planner will attempt to plan
+# a motion from the current position of the group
+# to the first point of the process path
```

Comment:
I will clarify that it's from the current joint pose to the first point in the trajectory.